<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bracingGroup</class>
 <widget class="QDialog" name="bracingGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="BracingGroupLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Edit Parameter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="addBracingGroupButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="deleteBracingGroupButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="bracingGroupTab">
          <attribute name="title">
           <string>Bracing Group</string>
          </attribute>
          <widget class="QTableWidget" name="bracingGroupTable">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>231</width>
             <height>391</height>
            </rect>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustIgnored</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Bracing Group</string>
            </property>
           </column>
          </widget>
         </widget>
         <widget class="QWidget" name="memberGroupTab">
          <attribute name="title">
           <string>Member Group</string>
          </attribute>
          <widget class="QTableWidget" name="memberGroupTable">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>231</width>
             <height>391</height>
            </rect>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Member Group</string>
            </property>
           </column>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="BGIterationLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Define Variable</string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="BGNameLabel">
           <property name="text">
            <string>Group Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="BGNameEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="IterationsLabel">
           <property name="text">
            <string>Variable Iterations</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="addBGIterationButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="deleteBGIterationButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="BGIterationTable">
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Bracing or Section Name</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateBGIterationButton">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bracingGroupButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
